docs/how-tos/python_consume.md

```diff
@@ -0,0 +1,105 @@
+# Python Consumer
+
+## Preface 
+
+This guide will explain how to fetch data from a GraphQL endpoint and map the response to a 
+Pydantic model. 
+We will cover: 
+
+- Defining a Pydantic model in [Define the Pydantic Model](#define-the-pydantic-model)
+- Fetching the data from GraphQL API in [Fetch Data from GraphQL API](#fetch-data-from-graphql-api)
+- Mapping the response to the Pydantic model in [Map Response Pydantic Model](#map-response-to-pydantic-model)
+
+!!! note "Alternative Options"
+
+    You may want to investigate the following libraries depending on your specific needs:
+      - [`gql`](https://github.com/graphql-python/gql) if you require support for subscriptions
+      - [`sgqlc`](https://github.com/profusion/sgqlc) for typed code generation
+      - [`ariadne-codegen`](https://github.com/mirumee/ariadne-codegen) for `pydantic` compatible code generation
+
+## Dependencies 
+
+This guide will utilize the following dependencies:
+
+- `requests` to send HTTP request to the GraphQL endpoint
+- `pydantic` for validating and parsing the fetched data
+
+## Define the Pydantic Model
+
+??? tip "More about Pydantic" 
+
+    Pydantic is a data validation and parsing library in Python. It provides a efficient way to 
+    validate and parse data into Python objects. We can use pydantic models to define the structure and type
+    of data we intend to fetch from the endpoint. It automatically validates the input data based on 
+    the defined type annotations. If the data does not match the expected types, Pydantic raises errors 
+    ensuring the integrity of the data. 
+
+To defind a Pydantic model, we define a class with Pydantic's `BaseModel` class as the base class. 
```

Comment:
Probably don't want to repeat `define`, and the first one was misspelled :stuck_out_tongue:
```suggestion
To create a Pydantic model, we define a class with Pydantic's `BaseModel` class as the base class. 
```